packages/ui/src/styles/fonts.css

```diff
@@ -203,51 +203,3 @@
   unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02C6, U+02DA, U+02DC,
     U+2000-206F, U+2074, U+20AC, U+2212, U+2215, U+E0FF, U+EFFD, U+F000;
 }
-/* roboto-300 - latin */
-@font-face {
```

Comment:
why do we remove the fonts?

Reply:
Generally, I recommend using system iOS/mac system font, as it supports all of the required chars, and it is the most consistent with the system UI. We can still provide our fonts. However, the definitions for the Roboto are broken, and for example, polish letters do not work, and they are taken from Open Sans, which looks ughly.